galsim/interpolatedimage.py

```diff
@@ -54,13 +54,20 @@ class InterpolatedImage(GSObject):
        multiplied by the pixel area.
 
     You can also use images that were drawn with one of the pixel-integrating methods ('auto',
-    'fft', or 'real_space'); however, the resulting profile will not correspond to the one
-    that was used to call `GSObject.drawImage`.  The integration over the pixel is equivalent to
-    convolving the original profile by a `Pixel` and then drawing with ``method='no_pixel'``.  So
-    if you use such an image with InterpolatedImage, the resulting profile will include the `Pixel`
-    convolution already.  As such, if you use it as a PSF for example, then the final objects
-    convolved by this PSF will already include the pixel convolution, so you should draw them
-    using ``method='no_pixel'``.
+    'phot', 'fft', or 'real_space'), or real images where nature has integrated over the pixel for
```

Comment:
I feel like one key point that the updated dox omits is that the need to do `no_pixel` prevents the use of photon shooting, so it can have serious impacts on computational cost -- and the existence of the `depixelize` option fixes that issue (at least, assuming the depixelize process is not too costly).  This point is implicitly there but I think it's valuable to mention explicitly.